services/github/github-common-fetch.js

```diff
@@ -24,9 +24,9 @@ const contentSchema = Joi.object({
   encoding: Joi.equal('base64').required(),
 }).required()
 
-async function fetchJsonFromRepo(
+async function fetchRepoContent(
   serviceInstance,
-  { schema, user, repo, branch = 'master', filename }
+  { user, repo, branch = 'master', filename }
```

Comment:
I hadn't realized that `Pipfile` is TOML but `Pipfile.lock` is JSON, so I unnecessarily refactored this. However it'd help with #3960 and https://github.com/badges/shields/issues/4068#issuecomment-534805506 so seems worth keeping.

Reply:
Nice!